src/task.c

```diff
@@ -44,6 +44,8 @@ u8 CreateTask(TaskFunc func, u8 priority)
         }
     }
 
+    // Error: Too many tasks!
+    // FIXME: This silently fails.
```

Comment:
This is supposed to silently fail. In the gen 3 debug ROM (specifically german ruby) it actually calls the crash function here instead of returning a 0.